packages/behandling-felles/src/util/readOnlyUtils.ts

```diff
@@ -16,13 +17,22 @@ const erReadOnly = (
   navAnsatt: NavAnsatt,
   fagsak: FagsakInfo,
   hasFetchError: boolean,
+  stegkode?: string,
+  klagevurdering?: KlageVurdering
 ) => {
   const { behandlingPaaVent } = behandling;
   const rettigheter = getRettigheter(navAnsatt, fagsak, behandling);
   const isBehandlingReadOnly = hasFetchError || harBehandlingReadOnlyStatus(behandling);
   const hasInactiveAksjonspunkter = aksjonspunkterForPunkt.some(ap => !ap.erAktivt);
+  const erVedtaksinstansstegForKlageSomHarKommetTilKlageinstans =
```

Comment:
Skurrer for meg at `behandling-felles` trenger å være avhengig av noe i `behandling-klage` for å utlede readonly.  Navngivningen av `erVedtaksinstansstegForKlageSomHarKommetTilKlageinstans` får det også til å virke som at dette løser noe som er veldig spesifikt for ett problem. 

Kan denne logikken flyttes ned på `behandling-klage`-nivå? Hvis readOnly-logikken trenger å utvides i behandling-klage, kan den utvides der istedetfor i en fellesmodul?

Reply:
I utgangspunktet valgte jeg å gjøre det på denne måten fordi det ville blitt forvirrende om `erReadOnly` ikke ga et endelig svar på hvorvidt steget er redigerbart eller ikke. Jeg kan godt se om det kan løses på en annen måte, men tror det blir vanskelig å gå helt utenom `behandling-felles` (i hvert fall `byggProsessmenySteg`).

Reply:
Har en løsning nå, skal bare teste litt før jeg pusher den ut her.

Reply:
Hvis dere trenger å håndtere `readOnly` i klage annerledes enn hva appen gjør ellers, så bør det håndteres i en klage-spesifikk modul så langt det er mulig - da avgrenses det til problemet dere trenger å løse, og logikken for å sette readOnly blir ikke mer komplisert for alle andre behandlingstyper eller ytelsestyper (hvis dette gjelder kun for f.eks. frisinn) som avhenger av samme logikk. Hvis det betyr at dere må override `readOnly` som sendes nedover fra f.eks. `KlagevurderingProsessIndex`, tenker jeg at det er en god trade-off for å slippe å påvirke masse annet. Huk tak i meg på Slack hvis jeg ikke forklarer hva jeg mener godt nok eller hvis jeg kan bistå med noe. 🙂 

Av nysgjerrighet - hva er det som gjør at readOnly-logikken som fins idag ikke holder i dette usecaset?

Reply:
Ser nå at du har fikset det, supert! Tar en ny titt

Reply:
> Av nysgjerrighet - hva er det som gjør at readOnly-logikken som fins idag ikke holder i dette usecaset?

Det er fordi klage behandles i to trinn - først av vedtaksinstans, deretter klageinstans. Vi vil at stegene som tilhører vedtaksinstans skal bli readonly når behandlingen kommer til klageinstans.